apps/mobile/src/components/widgets/collectibles/components/collectible-card-layout.tsx

```diff
@@ -0,0 +1,14 @@
+import { HasChildren } from '@/utils/types';
```

Comment:
Do we need a shared `types` package? 

We have this type here and also in the `extension`

Reply:
MIght make sense if we have quite a few that are the same, which I bet is the case ...especially once we start to refactor the extension after mobile.

Reply:
I did this in a new PR https://github.com/leather-io/mono/pull/481 as it wouldn't let me reopen this